internal/slicer/report.go

```diff
@@ -0,0 +1,61 @@
+package slicer
```

Comment:
Should this be in a new package itself? Maybe `db`?

Reply:
I am not sure because `// Report holds the information about files created when slicing packages.`. I thought about moving it to a different package but then the only use of `db` will be tightly coupled with the slicer. I was thinking on the following terms:
1. We are creating a report while we are slicing packages.
2. The report can be used to check for errors (e.g. several slices writing to the same file).
3. It happens that one of the ouputs is the chisel.db file.

Because the report has some *validation* usage inside slicer, I thought about keeping it in the same package. After my explanation, do you think we should move it?

Reply:
I understand your point. I am still a bit unsure if we should move it to some other package, but let's keep it as it is for now. We can always re-ogranise later if we feel the need.